templates/default/haproxy.cfg.erb

```diff
@@ -255,7 +255,9 @@ listen <%= key %>
   stats <%= option %> <%= value %>
 <% end -%>
 <% if listen['http_request'] -%>
-  http-request <%= listen['http_request'] %>
+<% listen['http_request'].each do |http_request| %>
```

Comment:
I don't think this works if `http-request` is a string? Because `string` doesn't have an `each` method.

Reply:
Ah, you are correct! Yeah, it looks fine to me. It would be cool to add some more unit tests - someday